packages/profile-sync-controller/src/controllers/user-storage/UserStorageController.ts

```diff
@@ -678,6 +679,28 @@ export default class UserStorageController extends BaseController<
     });
   }
 
+  /**
+   * Allows deletion of user data. Developers can extend the entry path and entry name through the `schema.ts` file.
+   *
+   * @param path - string in the form of `${feature}.${key}` that matches schema
+   * @returns nothing. NOTE that an error is thrown if fails to delete data.
+   */
+  public async performDeleteStorage(
+    path: UserStoragePathWithFeatureAndKey,
+  ): Promise<void> {
+    this.#assertProfileSyncingEnabled();
+
+    const { bearerToken, storageKey } =
+      await this.#getStorageKeyAndBearerToken();
+
+    await deleteUserStorage({
+      path,
+      bearerToken,
+      storageKey,
+      nativeScryptCrypto: this.#nativeScryptCrypto,
```

Comment:
Why are you passing `nativeScryptoCrypto` here if this isn't used by `deleteUserStorage` since no encryption/decryption takes place on this process.